CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Changelog
 
+## `v10.5.0`
+
+### Critical Changes
+
+1. This release fixes a critical issue for LDAP users, installing a needed package on Debian 11
+   on build-time. Moreover, a race-condition was eliminated ([#2341](https://github.com/docker-mailserver/docker-mailserver/pull/2341)).
+2. A resource leak in `check-for-changes.sh` was fixed ([#2401](https://github.com/docker-mailserver/docker-mailserver/pull/2401))
+
+### Other Major Changes
```

Comment:
```suggestion
### Other Minor Changes
```

Reply:
solved in [21d6b2c](https://github.com/docker-mailserver/docker-mailserver/pull/2431/commits/21d6b2c8fe9f8dff77b666ce434a360a938f8bc6)